### ISSUE TITLE
Stable/LTS Branch

### ISSUE THREAD

**Issue by dariusc93** [rejectedsoftware/vibe.d#1338]
As development continues with vibe, i feel there should be a stable branch that would be good for those who dont need to continue to upgrade, but continue to receive bug and security fixes during the lifetime of that release. Any thoughts on this? It doesnt even have to exactly be a branch but another repo where it can be maintain.

**Comment by s-ludwig** [comment]
The project is indeed approaching such a milestone. What I plan to do is to separate a few library parts into separate repositories and then use independent version numbers. Once that is done, stable branches for each part will be created. I don't know exactly when this will happen, but it's planned for directly after the next release.

**Comment by wilzbach** [comment]
Is there any progress on this? I do think that it would be nice to have a "small" core vibed and make stuff like redis or mongo as optional dependencies. 
That's basically the same development/evolution that express (from NodeJS) went through ;-)

**Comment by wilzbach** [comment]
Update: @s-ludwig I saw that the namespace `vibed` on github was still free and reserved it. I sent you an invitation as owner and will remove myself once you accept.

In case there no business considerations, I could imagine myself sth. like

vibed/mongo
vibed/redis
....

**Comment by s-ludwig** [comment]
See https://github.com/vibe-d - this is, step by step, receiving a revised version of the library, which will be tagged 1.0.0 once stable. The plan is to make the transition seamless in the sense that each component can be sourced from either the old or the new repository for a while.
